go.mod

```diff
@@ -184,6 +184,6 @@ replace github.com/google/flatbuffers => github.com/google/flatbuffers v1.11.0
 
 replace github.com/dgraph-io/ristretto => github.com/dgraph-io/ristretto v0.1.1-0.20211108053508-297c39e6640f
 
-replace github.com/bloxapp/ssv-spec => github.com/bloxapp/ssv-spec v0.0.0-20230521155754-79016edc0974 // #mev-capella
+replace github.com/bloxapp/ssv-spec => github.com/nkryuchkov/ssv-spec v0.0.0-20230523123751-7a59c345f536 // network-configs
```

Comment:
if we keep it diverged from main spec branch its better to push to spec's remote origin. though lets wait with it before we merge.